src/Footer/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import "./Footer.scss";
+import netiflyDark from "./images/netiflyDark.svg";
+
+class Footer extends React.Component {
```

Comment:
I would use a stateless functional component here instead of extending a Rea ct.Component because it's quite simple.

Reply:
“React: Component Class vs Stateless Component” @ven_korolyov https://itnext.io/react-component-class-vs-stateless-component-e3797c7d23ab